SpatialGDK/Source/SpatialGDK/Public/SpatialView/ConnectionHandler/SpatialOSConnectionHandler.h

```diff
@@ -29,6 +30,7 @@ class SpatialOSConnectionHandler : public AbstractConnectionHandler
 	TMap<int64, int64> InternalToUserRequestId;
 	FString WorkerId;
 	TArray<FString> WorkerAttributes;
+	SpatialEventTracer* EventTracer; // TODO: Figure out ownership constraints
```

Comment:
What does this TODO mean?

Reply:
Not sure actually. @improbable-danny do you know?

Reply:
Nope!